src/EMS/RingdownStatus.js

```diff
@@ -7,10 +7,17 @@ import Alert from '../Components/Alert';
 
 function RingdownStatus({ className, onStatusChange, ringdown }) {
   const [showUpdate, setShowUpdate] = useState(false);
+  const [showRedirect, setShowRedirect] = useState(false);
   function handleDiversionUpdate() {
```

Comment:
This function is related to Cancel, not Diversion, can we rename this function and its usage below?